com.unity.ml-agents/Runtime/Communicator/RpcCommunicator.cs

```diff
@@ -99,6 +100,41 @@ public RpcCommunicator(CommunicatorInitParameters communicatorInitParameters)
         }
 
         internal static bool CheckPythonPackageVersionIsCompatible(string pythonLibraryVersion)
```

Comment:
The diff is messy; the old code was renamed to CheckPythonPackageVersionIs*Supported*.